.env.test

```diff
@@ -0,0 +1,3 @@
+ROLLBAR_POST_SERVER_ITEM_ACCESS_TOKEN=
+ROLLBAR_POST_CLIENT_ITEM_ACCESS_TOKEN=
```

Comment:
I think we can remove these.
Won't be using rollbar for test environment probably.